packages/c/capnproto/xmake.lua

```diff
@@ -26,7 +26,20 @@ package("capnproto")
     end
 
     add_deps("cmake", "zlib")
-    on_install("windows|x86_64", "windows|i386", "linux", "macosx", "bsd", function (package)
+
+    -- Add on_check to fail for version greater than 1.0.2 on Windows ARM and MingW
+    on_check(function (package)
+        if package:version() >= "1.0.2" then
+            if is_plat("windows") and is_arch("arm*") then
+                raise("package(capnproto): Unsupported version on Windows ARM. Version 0.9.0 is the latest supported version on Windows ARM.")
+            end
+            if is_plat("mingw") then
```

Comment:
and here

Reply:
`package:` is done in 62f51cc33ce311a068e8c736b342db76873631d8